tests/mobly/asserts_test.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google Inc.
+# Copyright 2021 Google Inc.
```

Comment:
this is actually meant to stay at the initial year.
same for the other one

Reply:
Done.